extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -37,7 +39,8 @@
     @BuildStep
     public void build(ApplicationInfoBuildItem applicationInfo,
             KubernetesConfig kubernetesConfig,
-            List<KubernetesPortBuildItem> kubernetesPortBuildItems) throws UnsupportedEncodingException {
+            List<KubernetesPortBuildItem> kubernetesPortBuildItems,
+            Optional<KubernetesHealthPathItem> kubernetesHealthPathItem) throws UnsupportedEncodingException {
```

Comment:
As found by @gsmet , this should be `KubernetesHealthPathBuildItem` instead of `KubernetesHealthPathItem`